### PR TITLE
Update django-contrib-comments

### PR DESCRIPTION
**Pull request by vkaracic**
Version 1.8.0 of django-contrib-comments moved away from
using the SITE_ID settings attribute in favor of supporting
multi-tenancy.

LEARNER-1583 + EDUCATOR-733

@edx/helio and @awais786 please review.

### CODE REVIEW

**Review by clintonb — CHANGES_REQUESTED** [review]
Test? The last time we merged this, it broke. How do we know the breakage has been addressed?

**Comment by vkaracic** [comment]
@clintonb test added!

File: course_discovery/apps/publisher_comments/tests/test_forms.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+from django.test import TestCase
+
+from course_discovery.apps.publisher.tests.factories import CourseRunFactory
+from course_discovery.apps.publisher_comments.forms import CommentsForm
+from course_discovery.apps.publisher_comments.models import Comments
+
+
+class CommentsFormTests(TestCase):
+    """Tests for the CommentsForm form."""
```
Comment (clintonb) [inline course_discovery/apps/publisher_comments/tests/test_forms.py]: I've always found these class-level comments worthless. Feel free to remove this.

File: course_discovery/apps/publisher_comments/tests/test_forms.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        data = {
+            'name': 'Tester',
+            'email': 'tester@example.com',
+            'comment': 'Test comment'
+        }
+        data.update(f.initial)
+        return data
+
+    def test_get_comment_model(self):
+        """Retrieves the Comments model."""
```
Comment (clintonb) [inline course_discovery/apps/publisher_comments/tests/test_forms.py]: What retrieves the model? This doc string doesn't really help me understand what you're testing or why.

File: course_discovery/apps/publisher_comments/tests/test_forms.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        model = self.form.get_comment_model()
+        self.assertEqual(model, Comments)
+
+    def test_get_comment_create_data(self):
+        """Sets the right site_id value."""
```
Comment (clintonb) [inline course_discovery/apps/publisher_comments/tests/test_forms.py]: Same as above. Help me understand why this test is useful.

**Comment by vkaracic** [comment]
@clintonb I have somewhat addressed your comments. I've created a test on a more suitable place. The error that happens when upgrading this package happens when data is posted to the comment post endpoint hence this test. And I have confirmed that when I remove the `site_id` parameter this new test fails.

**Review by clintonb — APPROVED** [review]

**Pull request merged**